ui/pages/home/home.component.js

```diff
@@ -640,6 +651,53 @@ export default class Home extends PureComponent {
               defaultActiveTabName={defaultHomeActiveTabName}
               onTabClick={onTabClick}
               tabsClassName="home__tabs"
+              subHeader={
+                <Tooltip
+                  position="bottom"
+                  open={showPortfolioTooltip}
+                  interactive
+                  theme="home__subheader-link--tooltip"
+                  html={
+                    <div>
+                      <div className="home__subheader-link--tooltip-content-header">
+                        <div className="home__subheader-link--tooltip-content-header-text">
+                          {t('new')}
+                        </div>
+                        <button
+                          className="home__subheader-link--tooltip-content-header-button"
+                          onClick={() => {
+                            hidePortfolioTooltip();
```

Comment:
I think that `setPortfolioTooltipWasShownInThisSession` should be called whenever the tooltip is rendered, not just when it is closed. Or else it is redundant given the `showPortfolioTooltip` variable.